docs/source/examples/11/environment.yml

```diff
@@ -0,0 +1,12 @@
+name: ddp-env
+channels:
+  - defaults
+  - pytorch
+dependencies:
+  - pip=20.1.1
+  - python=3.7.3
+  - pytorch=1.8.0
+  - torchvision=0.9.0
+  - pip:
+    - hi-ml-azure>=0.1.0
+    - pytorch-lightning==1.3.8
```

Comment:
Can you upgrade ligyhtning to 1.5.2 please? Slightly different call syntax for the trainer though, take a look at my PR if needed